sympy/assumptions/handlers/order.py

```diff
@@ -215,10 +286,22 @@ def Basic(expr, assumptions):
             else:
                 return notpositive
 
+    @staticmethod
+    def Mul(expr, assumptions):
+        r = ask(Q.real(expr), assumptions)
+        if r is not True:
+            return r
+        if expr.is_number:
+            npos = fuzzy_not(AskPositiveHandler._number(expr, assumptions))
+            return npos
+        else:
+            npos = fuzzy_not(AskPositiveHandler.Mul(expr, assumptions))
+            return npos
```

Comment:
This seems unecessarily verbose,

```suggestion
        if expr.is_number:
            return fuzzy_not(AskPositiveHandler._number(expr, assumptions))
        else:
            return fuzzy_not(AskPositiveHandler.Mul(expr, assumptions))
```

Reply:
Done